lib/FetchEnvs.js

```diff
@@ -267,6 +289,27 @@ function reduceEnv(output, conf) {
   return output;
 }
 
+function reduceEnvSourceSimpleLinks(output, conf) {
+  const { valueFrom, parentNamespace  } = conf;
+  if (valueFrom !== undefined) {
+    let sourceValueFromInfo = Object.values(valueFrom)[0];
+    let sourceKind;
+    if( objectPath.get(sourceValueFromInfo, 'kind') ) //sourceValueFromInfo has a kind property, source is genericMapRef
+    {
+      sourceKind = objectPath.get(sourceValueFromInfo, 'kind');
+    }
+    else{ //else: sourceValueFromInfo doesn't have kind property, source is configMapRef or secretMapRef, use KIND_MAP to find which
+      sourceKind = KIND_MAP.get(Object.keys(valueFrom)[0]);
+    }
+    let sourceNamespace = sourceValueFromInfo.namespace || parentNamespace;
+    let sourceApiVersion = sourceValueFromInfo.apiVersion || 'v1';
+    let sourceSimpleLink = `${sourceApiVersion}:${sourceKind}/${sourceNamespace}/${sourceValueFromInfo.name}`;
+    let sourceData = {[sourceKind]: [sourceSimpleLink]};
```

Comment:
Do we need the key to be `[sourceKind]` rather than just `sourceKind`?

Reply:
if I remember right from my testing having just `sourceKind` with no brackets will use the string "sourceKind" as the knew instead of the value of sourceKind